### ISSUE TITLE
if i use stratum , i can't connect to the pool pls help!

### ISSUE THREAD

**Issue by undefined72** [oliverw/miningcore#1044]
hi  there

if  i use stratum  , i can't connect to pool . 

pls help me  to use stratum , i dont want use stratum2 .

**Comment by oliverw** [comment]
Excuse me, what?

**Comment by seperatis** [comment]
> hi there
> 
> if i use stratum , i can't connect to pool .
> 
> pls help me to use stratum , i dont want use stratum2 .

@undefined72 
I'm sorry, but **you do not understand**. Miner software like T-Rex that requires you to use `stratum2+tcp://` instead of `stratum+tcp://` is not actually USING **STRATUM PROTOCOL 2.0** as described [HERE](https://braiins.com/stratum-v2?utm_source=help): 

When you use `stratum2+tcp://` in your T-Rex miner config you are setting the Ethereum Stratum **Type**!

FROM THE T-REX README FILE:
```
 Supported schemes:                    stratum+tcp (This is Eth Proxy)
                                                      stratum+ssl
                                                      stratum+http
                                                      stratum2+tcp (This is Ethereum Stratum 1.0 / Nicehash)
                                                      stratum2+ssl
                                   stratum2 is normally used by Nicehash, MiningPoolHub and other similar mining pools
                                             Example:
                                            stratum2+tcp://daggerhashimoto.hk.nicehash.com:3353
```

tl;dr

Setting `stratum2+tcp://` in your T-Rex miner config **IS NOT** using Stratum Protocol V2, instead, it is telling the miner software to set the TYPE of **Ethereum Stratum Protocol** to "Ethereum Stratum 1.0 / Nicehash" instead of using Eth Proxy. Eth Proxy is not supported by Miningcore, currently, Miningcore only supports Ethereum Stratum 1.0 / Nicehash, therefore you must tell your miner software that the pool uses "Ethereum Stratum 1.0 / Nicehash", and the **WAY** in which you **DO** that with T-Rex is to use `stratum2+tcp://` in your config.

To do it with **Phoeneix Miner** you must set the **proto** flag like this `proto 4` as option 4 = Ethereum Stratum 1.0 / Nicehash per their documentation (see below)

```
-proto <n> Selects the kind of stratum protocol for the ethash pool:
     1: miner-proxy stratum spec (e.g. coinotron)
     2: eth-proxy (e.g. dwarfpool, nanopool) - this is the default, works for most pools
     3: qtminer (e.g. ethpool)
     4: EthereumStratum/1.0.0 (e.g. nicehash)
     5: EthereumStratum/2.0.0
```

I hope that helps you understand! Good Luck.

**Comment by oliverw** [comment]
Fixed by #350d05fc973c0a3f8912b3be062d4235821c9384
